Modix.Bot/Modules/PromotionsModule.cs

```diff
@@ -67,6 +67,37 @@ public async Task Campaigns()
             await ReplyAsync("", embed: embed);
         }
 
+        [Command("nominate")]
+        [Summary("Nominate the given user for promotion")]
+        public async Task Nominate(
+            [Summary("The user to nominate")]
+                IGuildUser subject,
+            [Remainder]
+            [Summary("A comment to be attached to the new campaign")]
+                string comment)
+        {
+            var continuationData = await PromotionsService.CreateCampaignAsync(subject.Id, comment);
+
+            var confirmationMessage = await ReplyAsync($@"You are nominating user {subject.Id} for promotion to rank {continuationData.TargetRankRole.Name}.
+React with ✅ in the next 10 seconds to finalize creation of the campaign.");
+
+            await confirmationMessage.AddReactionAsync(new Emoji("✅"));
+
+            await Task.Delay(10000);
+
+            var reactionUsers = await confirmationMessage.GetReactionUsersAsync("✅");
+
+            if (reactionUsers.Any(u => u.Id == continuationData.NominatingUserId))
+            {
+                await PromotionsService.ContinueCreateCampaignAsync(continuationData);
+            }
+            else
+            {
+                await confirmationMessage.RemoveAllReactionsAsync();
+                await confirmationMessage.AddReactionAsync(new Emoji("❌"));
```

Comment:
Just an idea - maybe it would be more clear to update the confirmation message's text to indicate that confirmation was not received in time and the campaign was not created.

Reply:
Definitely. The reactions make a lot more sense, I think, when we consider them to be a cheap knockoff for buttons. Which makes this message essentially a confirm dialog. We should add both reactions and see which one the user clicks. Then remove both of them and update/add a final result message

Reply:
Good idea. Implemented.